tools/appliance/systemvmtemplate/scripts/cleanup.sh

```diff
@@ -17,11 +17,10 @@
 # under the License.
 
 set -e
-set -x
 
 function cleanup_apt() {
   export DEBIAN_FRONTEND=noninteractive
-  apt-get -y remove --purge dictionaries-common busybox isc-dhcp-client isc-dhcp-common \
+  apt-get -y remove --purge dictionaries-common busybox \
```

Comment:
@Pearl1594 - can you check if there are anything else that could be purged ? (due to docker etc dependencies)